isis/src/system/objs/Kernel/Kernel.h

```diff
@@ -87,5 +87,17 @@ namespace Isis {
       QStringList m_kernels; //!< List of kernel file names
       Type m_kernelType;     //!< Enumeration value indicating the kernel type
   };
+
+  /**
+   * Logical operator for combining Type flags.
+   *
+   * @param a The first Type flag.
+   * @param b The second Type flag.
+   *
+   * @return Type flag that contains all Types in a and all Types in b.
+   */
+  inline Kernel::Type operator|(Kernel::Type a, Kernel::Type b) {
```

Comment:
My understanding is the "inline" is ignored by all modern compilers.

Reply:
yeah it does nothing. I'm going to pull it to make it cleaner. Our style-guide actually disallows it without testing right now.